components/style/build_gecko.rs

```diff
@@ -136,6 +136,16 @@ mod bindings {
             }
             if cfg!(target_os = "linux") {
                 builder = builder.clang_arg("-DOS_LINUX=1");
+            } else if cfg!(target_os = "solaris") {
+                builder = builder.clang_arg("-DOS_SOLARIS=1")
+            } else if cfg!(target_os = "dragonfly") {
+                builder = builder.clang_arg("-DOS_BSD=1").clang_arg("-DOS_DRAGONFLY=1")
+            } else if cfg!(target_os = "freebsd") {
+                builder = builder.clang_arg("-DOS_BSD=1").clang_arg("-DOS_FREEBSD=1")
```

Comment:
Terminating semicolons are missing but `cargo build` in Gecko didn't complain. I guess, it only affects style here.